tests/test_behavior.py

```diff
@@ -31,6 +31,7 @@ def non_dask_method(self, _dask_array_=None):
         return _dask_array_
 
 
+@pytest.mark.xfail(reason="NumPy 1.25 mixin __slots__ change")
```

Comment:
We could use a conditional fail (check `awkward>XXX || numpy>=YYY`), or just leave it until we fix it!

Reply:
thanks for the suggestion!